### PR TITLE
Add long term support policy

### PR DESCRIPTION
**Pull request by jescalan**
I don't think we need a description here, it is what it is!

### CODE REVIEW

File: docs/upgrade-guides/long-term-support.mdx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: Long Term Support Policy
+description: Clerk's policy for supporting past major versions of our SDKs
+---
+
```
Comment (LekoArts) [inline docs/upgrade-guides/long-term-support.mdx:5]:
```suggestion

# Long Term Support Policy

```

You need to also add the title to the markdown body

File: docs/upgrade-guides/long-term-support.mdx [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+When a new major version of an official Clerk SDK is released, the previous version is placed into long term support (LTS) mode for a period of *one year*. During this period, new features and bugfixes will not be actively backported, however, any security patches or other critical fixes will be actively backported.
```
Comment (LekoArts) [inline docs/upgrade-guides/long-term-support.mdx:6]:
```suggestion
When a new major version of an official Clerk SDK is released, the previous version is placed into long term support (LTS) mode for a period of **one year**. During this period, new features and bugfixes will not be actively backported, however, any security patches or other critical fixes will be actively backported.
```

I think making it bold is better for highlighting it

File: docs/upgrade-guides/long-term-support.mdx [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+After the LTS period ends, there will be no further changes to the code for that version in any form, and we will strongly recommend upgrading to a new version for product and security improvements. While we strongly recommend upgrading to keep your application secure, all previous versions of SDKs will continue to function indefinitely even if not upgraded.
```
Comment (LekoArts) [inline docs/upgrade-guides/long-term-support.mdx:8]: Once we release v5 we should add a table with dates

File: docs/upgrade-guides/long-term-support.mdx [hunk 4]
```diff
@@ -0,0 +1,8 @@
+---
+title: Long Term Support Policy
```
Comment (LekoArts) [inline docs/upgrade-guides/long-term-support.mdx:2]:
```suggestion
title: Long term support policy
```

Sentence case as per styleguide

**Review by dimkl — APPROVED** [review]
Nice work! I agree with the comments but overall i think it's good to be merged.

**Pull request merged**